applications/matter_weather_station/sample.yaml

```diff
@@ -19,8 +19,6 @@ tests:
   # ---------------
```

Comment:
should we add comment that this configuration cannot be build on twister integration plan, so be sure that this configuration is in quarantine

Reply:
OK, now we use test VID so we use default certs and chip-cert is no longer needed. We can test it in integration.